steps/pipeline-tests-jest.yaml

```diff
@@ -0,0 +1,43 @@
+parameters:
+- name: environment
+  default: ''
+- name: domain_prefix
+  default: ''
+- name: test_file
+  default: ''
+- name: workingDirectory
+  default: ''
+- name: junit_output_dir
+  default: ''
+- name: test_title
+  default: ''
+- name: root_domain
+  default: platform.hmcts.net
+
+steps:
+- task: Bash@3
+  displayName: Run pipeline tests
+  inputs:
+    targetType: 'inline'
+    script: |
+      mkdir -p ${{ parameters.junit_output_dir }}
+      yarn
+      yarn test
+    workingDirectory: '${{ parameters.workingDirectory }}'
+    failOnStderr: false
+  env:
+    JEST_JUNIT_OUTPUT_DIR: ${{ parameters.junit_output_dir }}
+    JEST_JUNIT_OUTPUT_NAME: report_$(system.jobId).xml
+    ${{ if eq(parameters.environment, 'prod') }}:
+      GLOBAL_DOMAIN_NAME: '${{ parameters.root_domain }}'
+    ${{ else }}:
+      GLOBAL_DOMAIN_NAME: '${{ parameters.domain_prefix }}.${{ parameters.root_domain }}'
```

Comment:
i'm not sure about this, 
i think it would be good to follow what the jenkins library does and provide a `TEST_URL` variable back to the tests.

the apps name should be passed in and a full URL set as an environment url.

(for the tests that want to check redirects that can be manipulated as needed)